README.md

```diff
@@ -1,10 +1,73 @@
-# grafana_blueflood_3.0.0
-Blueflood Plugin source code for Grafana (Using Graphite - API).
+# blueflood-grafana
 
-## Steps to install plugin
-cd blueflood-grafana-graphite_api-plugin-src
+Source code of the blueflood-grafana plugin to use [Blueflood](https://github.com/rackerlabs/blueflood) or 
+Rackspace-metrics as a datasouce with [grafana](http://grafana.org/). This plugin works with Grafana 3.0 and higher.
+
+
+## Pre-requisites
+
+* Running blueflood or Rackspace-metrics
+
+* This plugin requires graphite-api with blueflood finder as a service. You can setup this by running the 
+[rackerlabs/graphite-api-blueflood-finder](https://hub.docker.com/r/rackerlabs/graphite-api-blueflood-finder/) docker image as container.   
+
+```sh
+# graphite-api with blueflood finder
+    docker run \
+        -d -p 8888:8888 \
+        -e GRAFANA_URLS=http://localhost:3000,http://192.168.1.200:3000 \
+        -e BLUEFLOOD_QUERY_URL=http://localhost:20000\
+        -e TENANT_ID=123 \
+        rackerlabs/graphite-api-blueflood-finder
+```  
+
+* A grafana>=3.0 installation 
```

Comment:
replace ```grafana>=3.0``` with grafana 3.x. We don't know whether this will work with grafana 4.x?